src/tests/roc_pipeline/test_receiver.cpp

```diff
@@ -129,6 +129,65 @@ TEST(receiver, no_ports) {
     }
 }
 
+TEST(receiver, no_session_repair_port) {
```

Comment:
It seems that the first test is complete covered by the second test. I suggest to remove the first one.

Reply:
455a825